app/models/interest.rb

```diff
@@ -83,9 +83,9 @@ def summary
       target_type.underscore.to_sym.l + ": " +
       (target ? target.unique_format_name : "--")
   end
-  alias_method :text_name, :summary
+  alias text_name summary
```

Comment:
What is the difference between alias_method and alias?  Why is alias preferred here?  Is it always preferred to alias_method, or are there still cases where alias_method is used?